src/components/Content/TaskHierarchy/Task/Task.css

```diff
@@ -28,6 +29,15 @@
   cursor: pointer;
 }
 
+.new-form-btn {
+
+}
```

Comment:
@MartyTheeMartian This is a trivial comment on my part, but -- in the future how about do

```
.new-form-btn {
}
```

so that when we strictly add to it we'll just see green lines